src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt

```diff
@@ -95,7 +98,68 @@ class ReferenceTableRestServiceTest : BehaviorSpec({
                 JSONArray(responseBody).length() shouldBe 0
             }
         }
-        When("the get communicatiekanalen endpoint is called with 'true' as parameter") {
+        When("the communication channels reference table is retrieved") {
+            val response = itestHttpClient.performGetRequest(
+                "$ZAC_API_URI/referentietabellen/$communicationChannelPageReferenceTableId"
+            )
+            Then(
+                """the provisioned default communicatiekanalen are returned including 'E-formulier'"""
+            ) {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.isSuccessful shouldBe true
+                with(JSONObject(responseBody).toString()) {
```

Comment:
Do you think that `shouldEqualJson` would simplify this?
The problem would be `id` field that still needs schema validation :(

Reply:
yeah probably. I did not spent a lot of time on simplifying this test code to be honest.

Reply:
I will update the tests

Reply:
did this now. much better indeed :-)